ops_care.gemspec

```diff
@@ -29,4 +32,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler",  "~> 1"
   spec.add_development_dependency "rake",     "~> 10.0"
+  spec.add_development_dependency "reinteractive-style"
```

Comment:
This gem is included in every app that is hosted on OpsCare.
It needs to have the absolute minimal footprint, even for dev dependencies.
I rather it does not add the dependency on `reinteractive-style`.

Reply:
Thats fine will remove as a dependency